src/client/modules/profile/index.js

```diff
@@ -1,70 +1,98 @@
-var angular = require('angular');
-var ngRoute = require('angular-route');
-
+const angular = require('angular');
+const ngRoute = require('angular-route');
 module.exports = angular.module('tanks.profile', [
     ngRoute
 ]).config(RouteConfig)
-.service('profileService', ['$http', function ($http) {
-    var userId = '';
-// todo add norm function without param + change name
-    this.getProfileById = (id) => {//todo all changes to config
-        return $http.get("http://localhost:3000/api/users/profile/", id);
-    };
 
-    this.deleteAccount = () => {
-        return $http.put('http://localhost:3000/api/users/profile/delete/', {id: userId});
-    };
+    .service('profileService', ['$http', function ($http) {
+        let userId = '';
+        const profileURL = 'api/users/profile';
```

Comment:
Would b enice to move out every url svetysik ngs like this to separate file with consstants